### PR TITLE
feat(data): add CameraChannel.rtsps_quality mapping

### LINKED ISSUE #1005: Add CameraChannel.rtsps_quality mapping for RTSPS stream consumers
## Problem

`RTSPSStreams` is keyed by quality name (`high`/`medium`/`low`), but a consumer
iterating a camera's `CameraChannel`s has no library-provided way to know which
RTSPS quality a channel maps to. Consumers therefore hard-code the mapping — the
Home Assistant integration carries `{0: "high", 1: "medium", 2: "low"}` keyed on
`CameraChannel.id` — duplicating domain knowledge the library already owns on both
sides (`CameraChannel` and the `ChannelQuality` enum).

## Proposed solution

Add a read-only property `CameraChannel.rtsps_quality -> ChannelQuality | None`
mapping the channel to its RTSPS quality tier (id `0`→`HIGH`, `1`→`MEDIUM`,
`2`→`LOW`; `None` for non-standard ids). Consumers can then drop the hard-coded
dict and read `channel.rtsps_quality` directly.

Non-breaking (pure addition). The mapping lives where both models are defined,
which also nails down the channel↔quality semantics the library currently only
hints at (see the "naming appears inconsistent" note on `RTSPSStreams`).

## Additional context

Surfaced while migrating the Home Assistant `unifiprotect` integration's camera
streams onto the public API. Scoped deliberately to the channel→quality mapping
only: reading the library-primed `PublicCamera.rtsps_streams` stays a consumer
concern (a plain `public_bootstrap.cameras[id].rtsps_streams` lookup), and the
existing async `get_camera_rtsps_streams` live-fetch is unrelated — no second,
cached client method is wanted.

### PR DESCRIPTION
**Pull request by bluetoothbot**
## Description of change

Adds a read-only property `CameraChannel.rtsps_quality -> ChannelQuality | None` mapping a channel to its RTSPS quality tier (id `0`→`HIGH`, `1`→`MEDIUM`, `2`→`LOW`; `None` for non-standard ids). Consumers iterating a camera's channels can now read `channel.rtsps_quality` directly instead of hard-coding the mapping (the HA integration carries `{0: "high", 1: "medium", 2: "low"}`). Pure, non-breaking addition; lives where both `CameraChannel` and `ChannelQuality` are defined.

Closes https://github.com/uilibs/uiprotect/issues/1005

## Changes

- Add `CameraChannel.rtsps_quality` property backed by a module-level id→`ChannelQuality` map in `data/devices.py`.
- Add parametrized test covering ids 0/1/2 and non-standard ids (3, 4 → `None`).

## Test plan

- `poetry run pytest tests/data/test_camera.py -k "rtsps_quality or is_package"` — 11 passed.
- `ruff check` and `mypy` clean on changed files.

## Checklist

- [x] I have added tests
- [x] Code is formatted/linted

---
### Quality Report

**Changes**: 2 files changed, 42 insertions(+)

**Code scan**: clean

**Tests**: failed (FAILED)

**Branch hygiene**: clean

_Generated by [Kōan](https://koan.anantys.com)_

## Summary by CodeRabbit

* **New Features**
  * Camera channels now expose stream quality tier information for RTSP/RTSPS, mapping known channel IDs to quality levels and returning none when unavailable.
* **Tests**
  * Added coverage to verify the quality-tier mapping behavior across multiple channel IDs, including the no-mapping case.

### CODE REVIEW

**Review by RaHehl — DISMISSED** [review]

**Comment by RaHehl** [comment]
The mapping should also cover the package channel. `is_package` is `id == 3`, the server exposes a `package` RTSPS quality (per the `RTSPSStreams` docstring), and `ChannelQuality.PACKAGE` exists — so `id 3 -> None` is incomplete for a channel whose quality is known.

Please map `3: ChannelQuality.PACKAGE` (keep `None` for genuinely unknown ids, e.g. dual-sensor 180° channels), change the test param `(3, None)` -> `(3, ChannelQuality.PACKAGE)`, and update the docstring to `id 0->HIGH, 1->MEDIUM, 2->LOW, 3->PACKAGE`.

@bluetoothbot fix

File: tests/data/test_camera.py [hunk 1]
```diff
@@ -65,6 +66,34 @@ def test_camera_channel_is_package(channel_id: int, fps: int | None, expected: b
...
+        name="test",
+        enabled=True,
+        isRtspEnabled=False,
+        width=1920,
+        height=1080,
+        fps=15,
+        bitrate=1000,
+        fpsValues=[],
+        idrInterval=1,
+    )
```
[the remark this answers is not in the document]
Comment (RaHehl) [reply tests/data/test_camera.py:93]: Matches the adjacent `test_camera_channel_is_package`, which constructs the channel the same way. Both pin pure `channel.id`-based properties with no dependency on a captured firmware payload, so a `sample_data` fixture adds no fidelity here. Converting both id-based tests to fixtures would be a separate refactor, out of scope for this PR.

**Review by RaHehl — APPROVED** [review]
